swagger_spec_validator/validator20.py

```diff
@@ -196,6 +197,9 @@ def validate_apis(apis, deref):
     :raises: :py:class:`swagger_spec_validator.SwaggerValidationError`
     :raises: :py:class:`jsonschema.exceptions.ValidationError`
     """
+    operation_tag_to_operation_id_set = defaultdict(set)
+    # operation_ids = set()
```

Comment:
woopsies commented code, please delete

Reply:
Oh man you are so right I totally got distracted and forgot to check for silly stuff like this. My bad; I'll clean 'er up.